gcp-js-webserver/README.md

```diff
@@ -1,6 +1,6 @@
 [![Deploy](https://get.pulumi.com/new/button.svg)](https://app.pulumi.com/new)
 
-# Pulumi web server (GCP)
+# Web Server in Google Cloud
```

Comment:
Web Server Using Compute Engine? (I believe that's the name of "GCP's EC2"?)

Reply:
THANK YOU.